DTML.EduBot/Dialogs/UserInformation.cs

```diff
@@ -14,17 +14,21 @@ namespace DTML.EduBot.Dialogs
 {
     public partial class ChitChatDialog : QnaLuisDialog<object>
     {
-        [LuisIntent("UserInformation")]
+        [LuisIntent("UserInfo")]
         public async Task HandleUserInformation(IDialogContext context, LuisResult result)
         {
             EntityRecommendation entity;
             string BotResponse = BotPersonality.GetRandomGenericResponse() + Shared.ClientNewLine;
```

Comment:
I think this initialized value is never used, lets change to simply "string BotResponse;"